cmd/manager/main.go

```diff
@@ -183,9 +184,9 @@ func main() {
 	certsReady := make(chan struct{})
 	rotator := &mtls.CertRotator{
 		Ready:                         certsReady,
-		CAName:                        "tf-controller",
-		CAOrganization:                "weaveworks",
-		DNSName:                       "tf-controller",
+		CAName:                        "tofu-controller",
+		CAOrganization:                "flux-iac",
+		DNSName:                       "tofu-controller",
```

Comment:
Should be fine as long as the [upgrade guide](https://flux-iac.github.io/tofu-controller/use-tf-controller/upgrade-tf-controller/) is properly followed. Please correct me if I'm wrong.

Reply:
@chanwit do you have any inputs on this change?